build.ps1

```diff
@@ -1,10 +1,8 @@
 $ErrorActionPreference = "Stop"
 
-$testLogger = if ($env:GITHUB_ACTIONS -eq "true") {"GitHubActions;report-warnings=false"} else {"console"}
-
-dotnet test SentryNoSamples.slnf -c Release -l $testLogger `
+dotnet test SentryNoSamples.slnf -c Release -l "GitHubActions;report-warnings=false" `
     /p:CopyLocalLockFileAssemblies=true
 if ($LASTEXITCODE -ne 0) { exit 1 }
 
-dotnet pack SentryNoSamples.slnf -c Release /p:ContinuousIntegrationBuild=true
+dotnet pack SentryNoSamples.slnf -c Release --no-build /p:ContinuousIntegrationBuild=true
```

Comment:
Do we even need to pack via this script at all now?

Reply:
i assume if someone is using the script to build, they might also want the nugets for local testing?

Reply:
i will defer to you and @bruno-garcia on this. i am pretty comfortable with calling build/test/pack explicitly on the rare occasions i need to